src/App.js

```diff
@@ -1,27 +1,39 @@
-import logo from './logo.svg';
```

Comment:
- [ ] Your App.js file should be placed inside your components folder to comply with the project requirements, please do so 👍